app/src/androidTest/java/com/github/sdp/tarjetakuna/ui/webapi/WebApiViewModelTest.kt

```diff
@@ -246,7 +187,7 @@ class WebApiViewModelTest {
         }
 
         // call the api
-        viewModel.getCards()
+        viewModel.getRandomCard()
 
         // wait for the response - timeout of 1s, wait 3 times to be sure
         Utils.waitWhileTrue(1000, 3) { apiError.value == null }

```

Comment:
also, still missing the tests for cards_by_name, cards_by_set and set_by_code in view model.
Not sure if really needed, but i think so

Reply:
Completely true, I didn't have the courage to implement these tests as I'm particularly busy with my other task, and other courses. I will do them in another PR if required.